src/Repositories/CategoryRepository.php

```diff
@@ -24,13 +25,18 @@ public function get(int $categoryId, string $lang, array $with = [])
         $plentyCategoryRepo = pluginApp(PlentyCategoryRepositoryContract::class);
 
         $plentyCategory = $plentyCategoryRepo->get($categoryId, $lang);
+        $categoryDetailsList = $plentyCategory->details;
+        /** @var CategoryDetails $categoryDetails */
+        if($categoryDetailsList instanceof \ArrayAccess) {
+            $categoryDetails = $categoryDetailsList->offsetGet(0);
```

Comment:
Wenn dieser Fall nicht eintritt, knallt es aufgrund eines Zugriffs auf null in Zeile 39.

Reply:
Wenn dieser Fall nicht eintritt, hätte es schon vorher geknallt. Die Prüfung ist nur, um phan zu "überlisten".
Ich kann die Prüfung aber trotzdem einbauen